stages/org.osbuild.containers.unit.create

```diff
@@ -9,10 +9,11 @@ def validate(filename, cfg):
     # ensure the service name does not exceed maximum filename length
     if len(filename) > 255:
         raise ValueError(f"Error: the {filename} unit exceeds the maximum filename length.")
-
+    
```

Comment:
Didn't notice this at first but the linter caught it.  There's trailing whitespace here.